pyproject.toml

```diff
@@ -22,16 +22,17 @@ classifiers = [
 ]
 keywords = ["EEG", "MEG", "MNE", "GUI", "electrophysiology"]
 dependencies = [
-    "mne >= 1.7.0",
-    "PySide6 >= 6.7.1",
     "edfio >= 0.4.2",
     "matplotlib >= 3.8.0",
+    "mne >= 1.7.0",
     "numpy >= 2.0.0",
-    "scipy >= 1.14.1",
     "pybv >= 0.7.4",
+    "pyobjc-framework-cocoa >= 10.0; platform_system=='darwin'",
```

Comment:
```suggestion
    "pyobjc-framework-cocoa >= 10.0; platform_system=='Darwin'",
```

Reply:
The value of `platform_system` is case-sensitive and starts with an upper case letter, so this must not be changed.